src/System.Linq/tests/SkipWhileTests.cs

```diff
@@ -165,7 +165,7 @@ public int Current
         }
 
         [Fact]
-        [OuterLoop]
+        [ActiveIssue("Valid test but too intensive to enable even in OuterLoop")]
```

Comment:
I had that initially, but changed it because Skip results in yellow warning messages being printed to the console, something I'm trying to avoid for anything that's not actionable.  Skip is also confusing given how we've set up our ActiveIssue and OuterLoop constructs, because those tests are logically being skipped as well, yet the results will only report those actually marked as Skip'd.  We do have a couple of other test suites that use Skip, but they should really be ActiveIssue as well, and haven't changed yet due to some internal infrastructure issues.